packages/esm-ohri-pmtct-app/src/pmtct/patient-chart/banner-tags/banner-tags.component.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { usePatientOutcome } from './useInfantFinalOutcome';
+import { usePatientFamilyNames } from './usePatientFamilyNames';
+import { PatientStatusBannerTag } from '@ohri/openmrs-esm-ohri-commons-lib';
+
+interface MotherChildTagProps {
+  patientUuid: string;
+}
+
+const MotherChildTag: React.FC<MotherChildTagProps> = ({ patientUuid }) => {
+  const { t } = useTranslation();
+
+  const { patientOutcome } = usePatientOutcome(patientUuid);
+  const { childrenNames, motherName, patientGender, isLoading, isError } = usePatientFamilyNames(patientUuid);
+
+  if (isLoading) {
+    return null;
+  }
+
+  if (isError) {
+    console.error('Error fetching family information');
+    return null;
+  }
+
+  const outcomeColorMapping: { [key: string]: string } = {
+    'Still in Care': 'green',
+    'HIV negative infant discharged from PMTCT': 'green',
+    'Lost to followup': 'red',
+    Dead: 'red',
```

Comment:
How come ` Dead` is not a string

Reply:
Other outcomes have spaces, so wrapping them in quotes makes them valid string `keys.`

I have updated the `Dead` outcome for consistency.